Library/Formula/glew.rb

```diff
@@ -30,4 +20,8 @@ def install
     system "make", "GLEW_PREFIX=#{prefix}", "GLEW_DEST=#{prefix}", "all"
     system "make", "GLEW_PREFIX=#{prefix}", "GLEW_DEST=#{prefix}", "install.all"
   end
+
+  test do
+    assert_equal "GLEW version 1.12.0", shell_output("#{bin}/glewinfo | grep \"GLEW version\"").strip
```

Comment:
Let's not make this version specific. You can also just do:

`assert_match /#{version}/, shell_output("#{bin}/glewinfo")`

Which will do the same thing.